support-frontend/assets/pages/digital-subscription-checkout/components/endSummary/endSummary.jsx

```diff
@@ -10,21 +10,25 @@ const Dot = () => <div css={styles.dot} />;
 function EndSummary({ promotion, priceDescription }: EndSummaryProps) {
   return (
     <ul css={styles.list}>
+      <li>
+        <Dot /><div css={styles.listMain}>{promotion}</div>
+        <span css={styles.subText}>{priceDescription}</span>
+      </li>
       <li>
         <Dot /><div css={styles.listMain}>14-day free trial</div>
         <span css={styles.subText}>
           Your first payment will occur after the trial ends
         </span>
       </li>
-      <li>
-        <Dot /><div css={styles.listMain}>{promotion}</div>
-        <span css={styles.subText}>{priceDescription}</span>
-      </li>
       <li>
         <Dot /><div css={styles.listMain}>You can cancel any time</div>
       </li>
     </ul>
   );
 }
 
+EndSummary.defaultProps = {
+  endSummaryMobile: false,
```

Comment:
Is this `endSummaryMobile` prop necessary?

Reply:
No! Great pickup